frame/society/src/lib.rs

```diff
@@ -813,10 +814,12 @@ decl_module! {
 		/// Total Complexity: O(1)
 		/// # </weight>
 		#[weight = SimpleDispatchInfo::FixedNormal(10_000)]
-		fn found(origin, founder: T::AccountId) {
+		fn found(origin, founder: T::AccountId, max_members: u32) {
```

Comment:
Could also consider including a `pot` parameter which will bootstrap the pot, else we will wait 1 rotation period for it to grow.

If we include `pot`, we could also automatically transfer that amount from the founder to the Society account.

Reply:
the founder isn't meant to pay anything, and giving the `SetFounderOrigin` the capability of deducting as much as they want from any account in the system is obviously dumb.